Context/ContextInterface.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context;
+
+/**
+ * Stores the serialization or deserialization context (groups, version, ...).
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+interface ContextInterface
```

Comment:
Does https://github.com/Ener-Getick/FOSRestBundle/commit/6fa9d371452e4b61d9f450a4bab3313bc6a42c27 fix what you wanted ?